### PR TITLE
Fix menu and searchbar on tablet on classic

### LINKED ISSUE #26643: [FO] Header's searchbox on tablets is too big and off to the right
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

Header's searchbox on tablets is even bigger than on desktops. Also it leaks on the right.

obraz

On bigger screens it's ok:

obraz

### Expected behavior

_No response_

### Steps to reproduce

1. Open homepage on 768-991 px screen width.
2. Compare with other screen sizes

### PrestaShop version(s) where the bug happened

1.7.8.x

### PHP version(s) where the bug happened

_No response_

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Header layout is not ISO on menu and searchbar
| Type?             | bug fix
| Category?         | FO
| BC breaks?        | yes
| Deprecations?     | yes
| Fixed ticket?     | Fixes #26643
| How to test?      | See issue
| Possible impacts? | Menu header and searchbar

# BC Break
Because theme based on classic may have modified the header/menu

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26706)

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

**Comment by HanaRebaiQA** [comment]
Hello,

I have tested this PR and i noticed that the menu titles as well as searchbar are not aligned all together with logo. Look at the attached screenshots & screen-records below of PS1778x & PS1778 : 

PS178x : 
image

https://watch.screencastify.com/v/RnO9UvhjNCrx7xVKLziy

PS1778: 
image


https://watch.screencastify.com/v/XYgyQY96PBQwMnIPIzS4

Please check and feedback!

Thanks!

**Review by matks — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

File: themes/classic/_dev/css/components/mainmenu.scss [hunk 1]
```diff
@@ -26,9 +22,10 @@
 
   a[data-depth="0"] {
     padding: $small-space;
-    font-size: 1.125rem;
+    font-size: 1rem;
     font-weight: 600;
 
+
```
Comment (Progi1984) [inline themes/classic/_dev/css/components/mainmenu.scss]:
```suggestion
```

**Review by Progi1984 — COMMENTED** [review]
@NeOMakinG A small fix for removeing double empty lines

**Review by Progi1984 — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by HanaRebaiQA** [comment]
Hello @NeOMakinG 

It's QA :heavy_check_mark: 
The issue is fixed.
https://watch.screencastify.com/v/Yks8ZYDw9vaK2n8HaKad
image

Thanks!

**Comment by Progi1984** [comment]
Thanks @NeOMakinG & @HanaRebaiQA

**Pull request merged**